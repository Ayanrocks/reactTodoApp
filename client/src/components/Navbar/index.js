import React, { Component } from "react";

class Navbar extends Component {
  render() {
    return (
      <nav className="navbar fixed-top navbar-expand-md navbar-light bg-danger">
        <div className="navbar-brand text-light" href="#">
          React Todo App
        </div>
        <button
          className="navbar-toggler"
          type="button"
          data-toggle="collapse"
          data-target="#navbarNav"
          aria-controls="navbarNav"
          aria-expanded="false"
          aria-label="Toggle navigation"
        >
          <span className="navbar-toggler-icon"></span>
        </button>

        <div className="collapse navbar-collapse" id="navbarNav">
          <ul className="navbar-nav">
            <li className="nav-item active">
              <div className="nav-link text-light" href="#">
                Create
              </div>
            </li>
          </ul>
        </div>
      </nav>
    );
  }
}

export default Navbar;
