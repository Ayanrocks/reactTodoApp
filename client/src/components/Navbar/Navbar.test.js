import React from "react";
import Navbar from "./index";
import { BrowserRouter } from "react-router-dom";
import renderer from "react-test-renderer";

it("Navbar renders correctly", () => {
  const tree = renderer
    .create(
      <BrowserRouter>
        <Navbar />
      </BrowserRouter>
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
