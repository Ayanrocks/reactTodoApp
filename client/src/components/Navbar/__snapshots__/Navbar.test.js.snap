// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar renders correctly 1`] = `
<nav
  className="navbar fixed-top navbar-expand-md navbar-light bg-danger"
>
  <a
    className="navbar-brand text-light"
    href="/dashboard"
    onClick={[Function]}
  >
    React Todo App
  </a>
  <button
    aria-controls="navbarNav"
    aria-expanded="false"
    aria-label="Toggle navigation"
    className="navbar-toggler"
    data-target="#navbarNav"
    data-toggle="collapse"
    type="button"
  >
    <span
      className="navbar-toggler-icon"
    />
  </button>
</nav>
`;
